WORKSPACE

```diff
@@ -41,3 +41,16 @@ node_repositories(package_json = [
 # bazel run @yarn//:yarn
 # or
 # bazel run @nodejs//:npm
+local_repository(
+    name = "angular_devkit",
+    path = "../devkit",
+)
+git_repository(
+    name = "build_bazel_rules_typescript",
```

Comment:
Looks like a circular dependency? Is that the intention? I guess this will be mitigated with a monorepo at some point this year.

Reply:
Fixed now, this layering violation meant that the build-optimizer really needs to be in the angular repo.